src/Table/Table.js

```diff
@@ -91,19 +88,10 @@ export default class Table extends React.Component {
       state.itemHeight == null &&
       refs.itemHeightContainer != null) {
       // Calculate content height only once and when node is ready
-      newState.itemHeight = DOMUtil.getComputedDimensions(
+      let itemHeight = DOMUtil.getComputedDimensions(
         React.findDOMNode(refs.itemHeightContainer).querySelector('tr')
       ).height;
-    }
-
-    if (state.viewportHeight == null) {
```

Comment:
don't need viewportHeight because table doesn't need to know how many visible items there are